t/output_macros/niceTables.pg

```diff
@@ -0,0 +1,452 @@
+DOCUMENT();       
+
+loadMacros(
+	"PGstandard.pl",
+	"PGML.pl",
+	"MathObjects.pl",
+	"niceTables.pl",
+    "PGlateximage.pl"
+
+);
+
+$drawing = createLaTeXImage();
+$drawing->texPackages([['xy','all']]);
+$drawing->BEGIN_LATEX_IMAGE
+\xymatrix{ A \ar[r] & B \ar[d] \\\\
+           D \ar[u] & C \ar[l] }
+END_LATEX_IMAGE
+
+$path = insertGraph($drawing);
+
+
+$lorem = "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.";
```

Comment:
This line is too long.  No line should exceed 120 characters.  In addition there is excessive white space between `C<DataTable()>` and the text after it.  There really only needs to be one space there.  Generally in POD I would like to see even shorter than 120 characters.  This is really the same as what you have said about problem text being to wide.  It detracts from readability.

There are several places below that have the same problem.  I realize this is from @pstaabp's pull request.   Sorry he did that.  He should know better.